src/Symmetric/Openssl.php

```diff
@@ -167,6 +167,14 @@ class Openssl implements SymmetricInterface
      */
     protected $tagSize = 16;
 
+    /**
+     * Supported algorithms
+     *
+     * @internal
```

Comment:
Let's please explain why we added this internal annotation, for posterity.

Reply:
You submitted this suggestion, so I obeyed.

Reply:
> Marking it `@internal` and documenting that it's only there for BC purposes

From https://github.com/laminas/laminas-crypt/issues/29#issuecomment-1446314365

Reply:
CI is still complaining about the incorrect annotations group here: perhaps it wants `@internal` after `@var`, can't remember.

A sentence explaining the scope would be a good idea, for posterity.

Reply:
What comment do you want for posterity? ^_^"

Reply:
Maybe `@internal this property was declared for compatibility with PHP 8.2, and is not supposed to be used directly, other than for BC reasons` or such?

Reply:
Did you see the CI failures though?

Reply:
How would you write this comment in two lines? I'm having trouble finding the correct writing convention.
`Incorrect annotations group.`

Reply:
My endorsement would be to try running `./vendor/bin/phpcbf` once, which should attempt to fix it for you.

Reply:
phpcbf (3.7.2) doesn't seem to be bothered by this comment... despite the 39 errors found...

Reply:
Fair enough: I'll take it to merge from here, will try various things :-)

Thanks meanwhile!